integration/experiment/uncore_frequency_sweep/gen_cpu_activity_constconfig_recommendation.py

```diff
@@ -108,17 +112,17 @@ def main(full_df, region_list):
     df_region_group = df.groupby('region')
 
     # Characterize MBM metrics using the uncore sensitive
-    # region (intensity_0)
+    # region (intensity_1)
     mem_bw_characterization = system_memory_bandwidth_characterization(df_region_group, region=region_list[0])
 
     # A multi-step approach is used.
-    # First analyze the most uncore sensitive
-    # region (intensity_0) to find the efficient
+    # First analyze the uncore sensitive
+    # region (intensity_1) to find the efficient
     # uncore frequency
     uncore_freq_recommendation = frequency_recommendation(df_region_group, region=region_list[0],
                                                            domain="UNCORE")
 
-    # Then analyze the most core senstivite region (intensity_32)
+    # Then analyze the core senstivite region (intensity_16)
```

Comment:
Not to change my review, just curious: how come the change from intensity_0 -> intensity_1 and intensity_32 -> intensity_16?

Reply:
intensity_32 & intensity_0 are the extreme cases that we seldom see in workloads, so using something slightly less compute bound and slightly less memory bound appears to provide a better picture of general operation when tested on a number of different processor families.  

Ultimately this may not matter overall, and the user still has the ability to specify any region pair they wish.